servers/rendering/renderer_rd/effects/copy_effects.cpp

```diff
@@ -500,6 +501,47 @@ void CopyEffects::copy_depth_to_rect_and_linearize(RID p_source_rd_texture, RID
 	RD::get_singleton()->compute_list_end();
 }
 
+void CopyEffects::copy_depth_to_rect_and_linearize_reversed(RID p_source_rd_texture, RID p_dest_texture, const Rect2i &p_rect, bool p_flip_y, float p_z_near, float p_z_far) {
```

Comment:
Similar to my other comment, we don't need a whole different version to reverse the depth. I would instead just add a boolean flag to the push constant and add a parameter to ``copy_depth_to_rect_and_linearize()`` that defaults to false like: ``p_invert = false``

Reply:
I was going back and forth pondering if I wanted to change the original signature or not and ended up just going with the new function. I'll change it to your suggestion instead.

On a side note, there are a few functions in "copy_effects.cpp" that are very similar with the exception for push constants and mode, might be worth a future refactor to maybe make a more generic function that takes in those two things as parameters instead to save the duplication of code.